packages/wp-story-editor/src/components/documentPane/publish/publish.js

```diff
@@ -414,6 +447,41 @@ function PublishPanel({ nameOverride }) {
             </LabelWrapper>
           </DropdownWrapper>
         </HighlightRow>
+        {improvedAutosaves && (
+          <RevisionsWrapper>
+            <LabelWrapper>
+              <Label>
+                <LabelIconWrapper>
+                  <Icons.History width={24} height={24} aria-hidden />
+                </LabelIconWrapper>
+                <RevisionsLabel>
+                  {sprintf(
+                    /* translators: %d: number of revisions. */
+                    _n(
+                      '%d Revision',
+                      '%d Revisions',
+                      revisionCount,
+                      'web-stories'
+                    ),
+                    sprintf('%s', revisionCount)
```

Comment:
```suggestion
                    revisionCount
```

This should be fine here.